docs/source/subcommands.md

```diff
@@ -1,38 +1,217 @@
-CLI Reference
+# CLI Reference
+
 ---
 
-## `latch cp`
+## `latch init`
+
+```shell-session
+$ latch init PACKAGE_NAME
+```
 
-Copies local files into Latch Console.
+This command can be used to generate an example workflow ([this workflow](https://genomebiology.biomedcentral.com/track/pdf/10.1186/gb-2009-10-3-r25.pdf), to be specific) for reference purposes. It has all of the [required elements](basics/what_is_a_workflow.md) for a workflow, and can also be used as boilerplate - simply replace the logic with your logic, adding more tasks as necessary.
 
+The parameter `PACKAGE_NAME` is the name of the directory that `latch init` will create and populate with template files. The resulting directory structure is as below:
```

Comment:
think we should switch all references to template with "example" re: feedback from multiple ambassadors

Reply:
template implies proper boilerplate and they were confused